test/evergreen.yml

```diff
@@ -2193,7 +2193,7 @@ tasks:
       - func: "fetch artifacts"
       - func: "unit test"
         vars:
-          unit_test_args: -v 2 --long
+          unit_test_args: -v 2 --extra-long
```

Comment:
Should we update the task name to indicate it's an "extra-long" unit test task, and distinguish it from other "long" unit test tasks?

Reply:
Good idea, addressed here: [034987c](https://github.com/wiredtiger/wiredtiger/pull/8957/commits/034987c691596cb85c27db0f0fb9229cd4bdf11d)